lib/irb/cmd/load.rb

```diff
@@ -11,7 +11,7 @@
 #
 
 require "irb/cmd/nop.rb"
```

Comment:
Intended miss? There are some other ones below too.

Reply:
Good catch :+1:
Not intented. My script was not looking for `*.rb`, I'll have to tweak it.